cpp/oneapi/dal/detail/error_messages.cpp

```diff
@@ -38,6 +38,11 @@ MSG(unsupported_device_type, "Requested device type is not supported")
 MSG(small_data_block, "Data block size is smaller than expected")
 MSG(invalid_data_block_size, "Invalid data block size")
 MSG(method_not_implemented, "Method is not implemented")
+MSG(fill_memory_error, "Fill memory function returns error code")
```

Comment:
What does error code means here? It would be confusing for the user as we do not use error codes. 

This error id is misleadingly used for `memcpy`, probably it should say "Unknown error during memory copying"